src/instructions/proctored_exam/ReadyToStartProctoredExamInstructions.jsx

```diff
@@ -1,31 +1,31 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 import { getConfig } from '@edx/frontend-platform';
 import { Button, Container, Spinner } from '@edx/paragon';
-import ExamStateContext from '../../context';
 import Footer from './Footer';
 
+import { getExamReviewPolicy, startProctoredExam } from '../../data';
+
 const ReadyToStartProctoredExamInstructions = () => {
-  const state = useContext(ExamStateContext);
-  const {
-    exam,
-    getExamReviewPolicy,
-    startProctoredExam,
-  } = state;
+  const { exam } = useSelector(state => state.specialExams);
   const { attempt, reviewPolicy } = exam;
+
+  const dispatch = useDispatch();
+
   const examDuration = attempt.total_time ? attempt.total_time : exam.total_time;
   const platformName = getConfig().SITE_NAME;
   const rulesUrl = getConfig().PROCTORED_EXAM_RULES_URL;
   const [beginExamClicked, setBeginExamClicked] = useState(false);
 
   useEffect(() => {
-    getExamReviewPolicy();
+    dispatch(getExamReviewPolicy());
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
I found it odd we didn't get a lint error here for missing `dispatch`. I did a little reading, and it seems like the reference to `dispatch` is [very unlikely to change](https://stackoverflow.com/a/67012349). Maybe some linters have caught up to this and no longer lint on this? Where did you see that linting error originally?

I ask because I'd prefer we're consistent about whether we have dispatch in the dependency array.

Reply:
I believe the reason we're not getting a lint error is because of the eslint-disable comment on the next line. I agree that we should be consistent, so I can add `dispatch` to the dependency array.